selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestGitHubRepository.java

```diff
@@ -115,6 +127,21 @@ private void createFile(Path pathToRootContentDirectory, Path pathToFile) throws
     }
   }
 
+  /**
+   * Change content of the file
+   *
+   * @param pathToFile path to specified file
+   * @param content content to change
+   * @throws IOException
+   */
+  public void changeFileContent(String pathToFile, String content) throws IOException {
+    ghRepo.getFileContent(String.format("/%s", pathToFile)).update(content, "change - " + content);
+  }
+
+  public void deleteFile(String pathToContent) throws IOException {
```

Comment:
It's about _pathToFile_, not to _pathToContent_.

Reply:
Correct according to the last review